seatunnel-core/seatunnel-core-flink-sql/src/main/java/org/apache/seatunnel/core/sql/classloader/CustomClassLoader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.core.sql.classloader;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Path;
+
+public class CustomClassLoader extends URLClassLoader {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CustomClassLoader.class);
+
+    public CustomClassLoader() {
+        super(new URL[0]);
+    }
+
+    public void addJar(Path jarPath) {
+        try {
+            this.addURL(jarPath.toUri().toURL());
+        } catch (MalformedURLException e) {
+            LOGGER.error("Failed to add jar to classloader. Jar: {}", jarPath, e);
```

Comment:
replied in the previous comment, this might not cause the program broken. 

If missing the jar will cause a problem, it will cause a ClassNotFound in Flink processing. So I just leave the problem to Flink :)

What do you think ? or any other suggestion. cc @ruanwenjun